### PR TITLE
feat: checkbox item

### PR DESCRIPTION
**Pull request by winniezheng**

### CODE REVIEW

File: src/screens/Home/index.tsx [hunk 1]
```diff
@@ -32,6 +35,7 @@ const HomeScreen: React.FC<HomeScreenProps> = ({ appName }) => {
           alert("my nem jeff");
```
Comment (goofygoobers) [inline src/screens/Home/index.tsx:43]: don't wanna say but should we remove?

Comment (winniezheng) [reply src/screens/Home/index.tsx:43]: haha they're just placeholders for showing our resusable components

Comment (winniezheng) [reply src/screens/Home/index.tsx:43]: i assume tomas will remove when he starts on the home component

**Review by chrisrabe — APPROVED** [review]
looks good. Made some tweaks with the text style

**Pull request merged**